addons/hw_drivers/iot_handlers/drivers/DisplayDriver_L.py

```diff
@@ -173,3 +217,42 @@ def get_iot_devices(self):
         } for device in iot_devices]
 
         return json.dumps({'iot_device_status': iot_device})
+
+    @http.route('/kiosk/display', type='json', auth='none', methods=['POST'])
+    def kiosk_display(self, pos_id, access_token):
+        if not pos_id or not access_token:
+            return json.dumps({'status': 'failed', 'message': 'No id or access_token provided'})
+
+        display: DisplayDriver = DisplayDriver.get_default_display()
+        if not display:
+            return json.dumps({'status': 'failed', 'message': 'No display found'})
+
+        uri = f'{helpers.get_odoo_server_url()}/pos-self/{pos_id}?access_token={access_token}'
+
+        display.set_orientation(Orientation.RIGHT)  # default orientation for kiosk mode
```

Comment:
you can call the method w\o a parameter, given that you declare the default value already

Reply:
it's the default orientation for kiosk mode, not the default param of the method: we want the screen to be landscape 90% of the time, but when in kiosk mode, the new default is portrait (`Orientation.RIGHT`)